### ISSUE TITLE
Template Custom SVG

### ISSUE THREAD

**Issue by xwolfde** [RRZE-Webteam/rrze-expo#16]
Es soll ein individuell anpassbares Template geben, bei dem man ein eigenes SVG hochladen kann. In diesem SVG werden dann die Bereiche wie in #2 beschrieben dynamisch ermittelt und in Custom Fields umgesetzt.
Der Unterschied ist jedoch, dass hierbei das SVG des Templates nicht vorgegeben ist, sondern individuell hochgeladen werden kann.

**Comment by cassandre** [comment]
Ich würde eher Bereiche namentlich vorgeben, die dann von den Custom Templates genutzt und frei angeordnet werden können. Umgekehrt fände ich es zu fehleranfällig. Würde ich auch in #2 so handhaben.

**Comment by cassandre** [comment]
Wir sollen den Nutzern Vorlagen als Grundlage für die eigene Gestaltung zur Verfügung stellen. Allerdings so, dass nur die dran kommen, die sich damit auskennen, da wir keine Kapazitäten für SVG-Support haben.

**Comment by xwolfde** [comment]
ja, die Default-Vorlagen sind ja in #2 definiert. Hier geht es darum, dass man abseits der auch ein eigenes SVG hochladen kann.
